test.html

```diff
@@ -0,0 +1,319 @@
+                            <div class="course__head--icon">
+                                <img src="https://assets.scaler.com/assets/academy/svg/advanced-653b7e9152a8597a3487d8a31d2a408948aeaff459ed5964027e5234df083dc8.svg.gz"
+                                    alt="Advanced">
+                            </div>
+                            <div class="course__head--name">
+                                Advanced
+                            </div>
+                            <label class="course__toggle" for="Toggle2">
+                                <input class="toggle__input" name="" type="checkbox" id="Toggle2">
+                                <div class="toggle__fill"></div>
+                            </label>
+                        </div>
+                    </div>
+                </div>
+                <div class="course__info">
+                    <div class="course__details course__detail--beginner">
+                        <div class="course__details--description">
+                            I know bare minimum coding. Starting from scratch.
+                        </div>
+                        <div class="course__details--status">
+                            Comming Soon!
+                        </div>
+                        <div class="course__details--duration">
+                            44 Weeks Course
+                        </div>
+
+
+                    </div>
+                    <div class="course__details course__detail--intermediate">
+                        <div class="course__details--description">
+                            I know coding in one programming language. Can build basic applications.
+                        </div>
+                        <div class="course__details--status">
+                            Enrolment open
+                        </div>
+                        <div class="course__details--duration">
+                            37 Weeks Course
+                        </div>
+                    </div>
+                    <div class="course__details course__detail--advanced">
+                        <div class="course__details--description">
+                            I am good at coding. I know basic problem solving, data structures, algorithms. Can build
+                            basic to advanced
+                            applications.
+                        </div>
+                        <div class="course__details--status">
+                            Enrolment open
+                        </div>
+                        <div class="course__details--duration">
+                            30 Weeks Course
+                        </div>
+                    </div>
+                </div>
+                <div class="course__content">
+                    <div class="content__syllabus syllabus--beginner">
+                        <div class="syllabus__bookmark bookmark--first">
```

Comment:
cannot give modifier class name without using the class first
```suggestion
                    <div class="content__syllabus syllabus syllabus--beginner">
                        <div class="syllabus__bookmark bookmark bookmark--first">
```